docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -219,6 +220,23 @@ $ tsh touchid diag
 # Touch ID enabled? true
 ```
 
+### Windows Webauthn not usable
+
+Windows Webauthn support for `tsh` requires Windows 10 19H1 or later.
```

Comment:
I think we should mention the OS requirement in the Prerequisites so users with unsupported Windows versions aren't disappointed.

Reply:
added